bin/common/versions.sh

```diff
@@ -57,3 +57,11 @@ then
     echo ubuntu '       =' $UBUNTU_VERSION
     echo
 fi
+
+if id -u vagrant >& /dev/null; then
+  SCF_BIN_DIR="${SCF_BIN_DIR:-/usr/local/bin}"
+  SCF_TOOLS_DIR="${SCF_TOOLS_DIR:-/home/vagrant/tools}"
+else
+  SCF_BIN_DIR="${SCF_BIN_DIR:-output/bin}"
+  SCF_TOOLS_DIR="${SCF_TOOLS_DIR:-output/tools}"
+fi
```

Comment:
This was actually based on your feedback, about *not* putting things in `/usr/local/bin` for users who might run the `install_tools` scripts locally. Presumably no vagrant user would exist in such cases (so it makes sense to put things in the scf repo)